.github/ISSUE_TEMPLATE/other-issue.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Other issue
+about: Use this issue template to describe a current issue with an analysis or documentation (that is not a new example analysis)
+title: 'Other issue'
+labels: other issue
```

Comment:
Similar question about the title - how does automatically labeling these with `other issue` help us?

Reply:
Probably doesn't. I looked at the openpbta issue templates and this type of things was there, but we can delete as many of these categories as is useful.

Reply:
That's a very different beast! These are supposed to help us on a project that will very much be internal. So in addition to what will result in good issues that have the required context and are actionable, we need to think about how we're going to _find stuff_ and get a high-level look at the state of the project.

Reply:
Yes I understand better what the intent is for these now. 👍